CodePushConfig.m

```diff
@@ -40,6 +50,7 @@ - (instancetype)init
                             buildVersion,BuildVdersionConfigKey,
                             serverURL,ServerURLConfigKey,
                             deploymentKey,DeploymentKeyConfigKey,
+                            clientUniqueId,ClientUniqueIDConfigKey,
```

Comment:
This needs to be above the deployment key entry since it can be nil (e.g. the dev didn't add a key to the info.plist file), and nil will signal the end of the key pairs being added to the dictionary.